### DISCUSSION TITLE
how to decrease app size

### DISCUSSION THREAD

**Discussion by wishrohitv** [espresso3389/pdfrx#210] [Q&A] [answered]
normal apk size is big about 29mb with 15 lines of code 🤔

**Comment by espresso3389** [comment, marked as answer]
Currently pdfrx Android supports 4 architectures, arm64-v8a, armeabi-v7a, x86, and x86_64 and every arch libpdfium.so binary consumes around 4MB; so 16MB is minimum. Not so many things I can do.

**Comment by wishrohitv** [comment]
ok thanks
